requirements-docs.txt

```diff
@@ -2,3 +2,4 @@ Sphinx>=1.3.1,!=1.7.1
 sphinx-rtd-theme
 sphinx-intl
 sphinx-reredirects
+pyyaml
```

Comment:
What relies on the `pyyaml` dependency?

Reply:
See https://github.com/natcap/invest.users-guide/commit/df9acf867db2882aae4c4f7c7103243160d5e4b1